restricted_site_access.php

```diff
@@ -215,12 +215,13 @@ public static function restrict_access( $wp ) {
 		}
 
 		// check for the allow list, if its empty block everything
-		if ( !empty( self::$rsa_options['allowed'] ) && is_array( self::$rsa_options['allowed'] ) ) {
+		if ( ! empty( self::$rsa_options['allowed'] ) && is_array( self::$rsa_options['allowed'] ) ) {
 			$remote_ip = get_client_ip_address(); //save the remote ip
 
 			// iterate through the allow list
 			foreach( self::$rsa_options['allowed'] as $line ) {
 				if( ip_in_range( $remote_ip, $line ) ){
+					do_action( 'restrict_site_access_ip_match', $remote_ip ); // allow users to hook ip match
```

Comment:
New actions should be documented using the [WordPress inline documentation standard](https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/php/#4-hooks-actions-and-filters).